examples/api/lib/cupertino/nav_bar/cupertino_sliver_nav_bar.0.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for CupertinoSliverNavigationBar
+
+import 'package:flutter/cupertino.dart';
+
+void main() => runApp(const MyApp());
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  static const String _title = 'CupertinoSliverNavigationBar Sample';
+
+  @override
+  Widget build(BuildContext context) {
+    return const CupertinoApp(
+      title: _title,
+      home: CupertinoNavBarSample(),
+    );
+  }
+}
+
+class CupertinoNavBarSample extends StatelessWidget {
+  const CupertinoNavBarSample({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return CupertinoPageScaffold(
+      // A ScrollView that creates custom scroll effects using slivers.
+      child: CustomScrollView(
+        // A list of sliver widgets.
+        slivers: <Widget>[
+          const CupertinoSliverNavigationBar(
+            leading: Icon(CupertinoIcons.person_2),
+            // This title is visible in both collapsed and expanded states.
+            // when `middle` parameter is omitted, it takes the `largeTitle`.
```

Comment:
```suggestion
            // When the "middle" parameter is omitted, it takes the "largeTitle".
```

And what do you mean by "takes"?  Uses?

Reply:
Yes

I  just updated this description like this to make it more clear:
```console
            // This title is visible in both collapsed and expanded states.
            // When the "middle" parameter is omitted, the widget provided
            // in the "largeTitle" parameter is used instead in the collapsed state.
```